src/fu-util.c

```diff
@@ -4395,12 +4399,14 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Switch the firmware branch on the device"),
 			      fu_util_switch_branch);
+#ifdef HAVE_HSI
 	fu_util_cmd_array_add(cmd_array,
```

Comment:
is this what we do for other commands? I think not hiding the argument and printing something like "is not supported" is probably more helpful.

Reply:
adjusted